backend/BackendMIIT/src/main/java/com/example/BackendMIIT/model/domain/Direction.java

```diff
@@ -12,12 +12,17 @@
 @Data
 public class Direction extends BaseEntity {
 
+    private String code;
     private String name;
-    private int passPoint;
+    private String level;
```

Comment:
What's this variable responsible for?

Reply:
It responds for grade level of speciality